centreon/src/Core/Host/Infrastructure/API/AddHost/AddHostSaasSchema.json

```diff
@@ -120,6 +120,24 @@
                 "type": "integer"
             }
         },
+        "event_handler_enabled": {
+          "type": [
+            "null",
+            "integer"
+          ],
+          "enum": [
+            null,
+            0,
+            1,
+            2
```

Comment:
Should this value be nullable ? According to the ticket the enabled status should be "Yes"/"No"/"Default"